test/end-to-end-tests/start.ts

```diff
@@ -33,6 +33,14 @@ program
 
 const hsUrl = 'http://localhost:5005';
 
+// XXX: why is this not located in @types/global.d.ts?
```

Comment:
references to window.mxPerformanceMonitor in this file don't seem to be using the definition from `@types/global.d.ts`

Reply:
I think creating a `@types/global.d.ts` relative to the src directory with an import of the react-sdk global.d.ts should do the trick, this is what element-web does.